_posts/2021-03-29-Encoding the Lending Library.md

```diff
@@ -0,0 +1,214 @@
+
+<!--more-->
+
+### Ellen Wright
+Through the Shakespeare and Company Project website, I selected the collection of [all cards associated with Ellen Wright](https://shakespeareandco.princeton.edu/members/wright-ellen/cards/0d89fdbb-a48a-4bfa-8690-c53234d02331/), an American living in Paris with her husband, the acclaimed African American author Richard Wright. There are two cards associated with Wright, pictured below:
+
+![Front of Card 1](https://iiif.princeton.edu/loris/figgy_prod/13%2Fe8%2F39%2F13e839f94ac344f7bc0a5793c5c4df80%2Fintermediate_file.jp2/full/450,/0/default.jpg)
+
+![Front of Card 2](https://iiif.princeton.edu/loris/figgy_prod/93%2Fdd%2F4d%2F93dd4d3a5fa048ed8e88685bd2df1e0c%2Fintermediate_file.jp2/full/450,/0/default.jpg)
+
+###### Image Credits: [Wright, Ellen; Sylvia Beach Papers, C0108, Manuscripts Division, Department of Special Collections, Princeton University Library](https://findingaids.princeton.edu/catalog/C0108_c3762)
+
+Though there is overlap between the information on the two cards, I chose to encode them both. I did not encode the backs of the cards, as the first card is blank on the reverse, and the other side of second card is an advertisement from the Princeton Bank and Trust Company.
+
+### Important Elements
+When encoding these cards, I began by identifying the key items that formed the basis of the borrowing records: the borrow date, the book title, and the return date. I also encoded the heading on the top of each card, which contained the library member's name and home address. Another key aspect of these cards were the other library members mentioned, many of whom were accomplished authors themselves, such as Richard Wright and Gertrude Stein. Finally, I noted that the handwritten text on each card changed color multiple times, a result of the process by which the card was updated over the span of months each time Wright borrowed or returned books. Though the particular colors used may not have much significance, they provide evidence for the ways the card changed over time. By capturing the key information about an individual's borrowing habits in this encoded form, it is possible to analyze this data more effectively and systematically compare Wright's records with those of other library members.
```

Comment:
I love how you've noted the changing colors of ink as material evidence of use over time. A smart move to encode those changes in XML.